lib/packaging/gem/lib/govuk_frontend_alpha.rb

```diff
@@ -0,0 +1,26 @@
+# Conventionally the engine lives in a separate file and imported here, i've
+# combined them to reduce the number of files while we work on packaging.
+
+module GovukFrontendAlpha
+  class Engine < ::Rails::Engine
+    initializer "govuk_frontend_alpha.assets.precompile" do |app|
+      app.config.assets.precompile += %w(
+        govuk-frontend*.css
```

Comment:
Would be nice if we could generate this list automatically, but it’s not needed for alpha.